packages/gatsby-theme-newrelic/src/components/OneTrust.js

```diff
@@ -1,40 +1,30 @@
 import React from 'react';
-import { graphql, useStaticQuery } from 'gatsby';
+import PropTypes from 'prop-types'
 
-const OneTrust = () => {
-  const {
-    site: {
-      siteMetadata: { oneTrustID },
-    },
-  } = useStaticQuery(graphql`
-    query {
-      site {
-        siteMetadata {
-          oneTrustID
-        }
-      }
-    }
-  `);
+const OneTrust = ({ id }) => {
```

Comment:
is `id` populated with the same value it was using previously -- `oneTrustID`?

Reply:
This is the component prop. I thought it might be a little wonky to say `<OneTrust oneTrustID={oneTrustID} />` but maybe it's better to be consistent.

Reply:
Oh, but I forgot to add the prop in. Just updated that.